docs/docs/user-guides/egress-rules.mdx

```diff
@@ -3,6 +3,8 @@ title: Egress Rules
 sidebar_position: 3
 ---
 
+# Access control (egress rules)
```

Comment:
`Access control` may be somewhat ambiguous -- admin access control or network access control?

```suggestion
# Network Access Control
```